addons/website/static/src/client_actions/configurator/configurator.xml

```diff
@@ -231,6 +231,29 @@
                             </t>
                         </div>
                     </div>
+                    <t t-if="state.showExtraThemesButton and !state.extraThemes.length and state.themes.length == nbrMainThemes">
```

Comment:
I am not sure to understand the logic here: why is not `!state.extraThemes.length` enough? Do we actually need `showExtraThemesButton`? And why the `state.themes.length == nbrMainThemes` ?

Reply:
- The `showExtraThemesButton` is for the edge case where there are only 3 themes to show. In this case, if the user clicks on the button, no extra themes are shown (`state.extraThemes.length ==0`) so I thought it would be better to hide the button in this case -> The user tries to load more themes but there are no available, no need to retry.
- The `state.themes.length == nbrMainThemes` is for a case where there are less than 3 available themes. In this case, it though it would be better to not even show the button as we already know that clicking on it would have no effect :+1:

Reply:
Create a property in your component which returns this condition and where you can add some documentation about this :+1: